src/components/LogRow/BaseRow/index.tsx

```diff
@@ -40,34 +40,35 @@ const BaseRow = forwardRef<any, BaseRowProps>((props, ref) => {
     children,
     "data-cy": dataCyText,
     index,
-    highlightedLine,
+    highlights,
     lineNumber,
     prettyPrint = false,
+    searchLine,
     searchTerm,
     resmokeRowColor,
     wrap,
     resetRowHeightAtIndex,
-    highlights,
     scrollToLine,
     ...rest
   } = props;
 
+  const { sendEvent } = useLogWindowAnalytics();
+
   const [shareLine, setShareLine] = useQueryParam<number | undefined>(
     QueryParams.ShareLine,
     undefined
   );
-  const shared = shareLine === lineNumber;
 
   const [bookmarks, setBookmarks] = useQueryParam<number[]>(
     QueryParams.Bookmarks,
     []
   );
-  const { sendEvent } = useLogWindowAnalytics();
-  const bookmarked = bookmarks.includes(lineNumber);
 
-  const highlighted = highlightedLine === index;
+  const shared = shareLine === lineNumber;
+  const bookmarked = bookmarks.includes(lineNumber);
+  const highlighted = searchLine === index;
 
-  // Clicking a line should select or deselect the line.
+  // Clicking link icon should set or unset the share line.
   const handleClick = () => {
     if (shared) {
       setShareLine(undefined);
```

Comment:
I can't select the line below this but wrapping handleDoubleClick with a `useCallback` prevents an entire render.